src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -1498,6 +1499,62 @@ static QVariant fcnToInterval( const QVariantList &values, const QgsExpressionCo
   return QVariant::fromValue( QgsExpressionUtils::getInterval( values.at( 0 ), parent ) );
 }
 
+/*
+ * DMS functions
+ */
+
+static QVariant floatToDegreeFormat( const QgsCoordinateFormatter::Format format, const QVariantList &values, const QgsExpressionContext *, QgsExpression *parent, const QgsExpressionNodeFunction * )
+{
+  double value = QgsExpressionUtils::getDoubleValue( values.at( 0 ), parent );
+  QString axe = QgsExpressionUtils::getStringValue( values.at( 1 ), parent );
+  int precision = QgsExpressionUtils::getIntValue( values.at( 2 ), parent );
+
+  QString format_string = QStringLiteral( "" );
```

Comment:
No need to initialize (in factor msvc will complain about `QStringLiteral( "" )`, just go

    QString formatString;

(also use camel case).